src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -404,12 +405,15 @@ def _compute_metric(self, waveform_extractor, metric_name, **metric_params):
             metric = metric_func(waveform_extractor, **metric_params)
         elif (metric_name == 'snr' or
               metric_name == 'peak_offset'):
-            peak_sign = metric_params['peak_sign']
-            del metric_params['peak_sign']
-            metric = metric_func(waveform_extractor,
-                                 peak_sign=peak_sign, **metric_params)
-        else:
-            metric = {str(unit_id): metric_func(waveform_extractor,
+            if 'peak_offset' in metric_params:
+                peak_sign = metric_params['peak_sign']
+                del metric_params['peak_sign']
+                metric = metric_func(waveform_extractor,
+                                    peak_sign=peak_sign, **metric_params)
+            elif metric_name == 'snr':
```

Comment:
I think `peak_sign` is needed as an argument for both `snr` and `peak_offset`, or at least an iteration of this had it as such, so the `elif` might be unnecessary. It also seems like line 408 might want to check for `peak_sign` instead of `peak_offset`? Wanted to double-check before making the change.